### PR TITLE
Merge 4.5.2 into 4.6.0

### PR DESCRIPTION
**Pull request by yenienserrano**
Merge 4.5.2 into 4.6.0

### CODE REVIEW

File: yarn.lock
Comment (Desvelao) [inline yarn.lock:1]: **nitpick**: remove this file.

File: scripts/release/tag.js
Comment (Desvelao) [inline scripts/release/tag.js:1]: **nitpick**: remove this file due this is in `plugins/main/scripts&release` folder.

File: scripts/release/lib/update-manifest-plugin.js
Comment (Desvelao) [inline scripts/release/lib/update-manifest-plugin.js:1]: **nitpick**: remove this file due this is in `plugins/main/scripts&release` folder.

File: scripts/release/lib/update-manifest-package.js
Comment (Desvelao) [inline scripts/release/lib/update-manifest-package.js:1]: **nitpick**: remove this file due this is in `plugins/main/scripts&release` folder.

File: scripts/release/lib/read-manifest-package.js
Comment (Desvelao) [inline scripts/release/lib/read-manifest-package.js:1]: **nitpick**: remove this file due this is in `plugins/main/scripts&release` folder.

File: scripts/release/bump.js
Comment (Desvelao) [inline scripts/release/bump.js:1]: **nitpick**: remove this file due this is in `plugins/main/scripts&release` folder.

**Review by Desvelao — DISMISSED** [review]
LGTM!

File: yarn.lock
Comment (yenienserrano) [reply yarn.lock:1]: Done

File: scripts/release/lib/update-manifest-plugin.js
Comment (yenienserrano) [reply scripts/release/lib/update-manifest-plugin.js:1]: Done

**Review by asteriscos — APPROVED** [review]
LGTM :heavy_check_mark:

File: scripts/release/tag.js
Comment (yenienserrano) [reply scripts/release/tag.js:1]: Done

File: scripts/release/lib/read-manifest-package.js
Comment (yenienserrano) [reply scripts/release/lib/read-manifest-package.js:1]: Done

File: scripts/release/lib/update-manifest-package.js
Comment (yenienserrano) [reply scripts/release/lib/update-manifest-package.js:1]: Done

File: scripts/release/lib/logger.js
[the remark this answers is not in the document]
Comment (yenienserrano) [reply scripts/release/lib/logger.js:1]: Done

File: scripts/release/bump.js
Comment (yenienserrano) [reply scripts/release/bump.js:1]: Done

**Review by Desvelao — APPROVED** [review]
LGTM!

**Pull request merged**